config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  mount MaintenanceTasks::Engine, at: "/maintenance_tasks"
```

Comment:
We need to restrict access to this path to admins.  @Oaphi do you have any idea how to do that?  (Pinging you since you suggested the package, so hoping you know more about it.)